packages/flet/lib/src/controls/card.dart

```diff
@@ -6,6 +7,8 @@ import '../utils/colors.dart';
 import '../utils/edge_insets.dart';
 import 'create_control.dart';
 
+enum CardVariant { normal, filled, outlined }
+
```

Comment:
Indeed, M3 spec has 3 types of card: https://m3.material.io/components/cards/overview - nice to have it in line with the spec!

Reply:
Ah, thanks!
I thought for a while on a better name for that "normal", but "elevated" didn't pass my mind. Haha!